types/k6/experimental/browser.d.ts

```diff
@@ -1131,6 +1148,37 @@ export interface ElementHandle extends JSHandle {
      */
     selectText(options?: ElementHandleOptions): void;
 
+    /**
+     * Sets the value of the file input to these files.
+     *
+     * To work with local file on the file system, work with the experimental
+     * fs module to load and read the file's content.
+     *
+     * The elementHandle must an [input element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input).
+     * @param files
+     * @param options
+     */
+    setInputFiles(files: File | File[], options?: {
+        /**
+         * Maximum time in milliseconds. Pass 0 to disable the timeout. Default
+         * is overridden by the setDefaultTimeout option on `BrowserContext` or
+         * `Page`. Defaults to 30000.
+         */
+        timeout?: number;
+
+        /**
+         * Setting this to `true` will bypass the actionability checks (visible,
```

Comment:
To be consistent with the `noWaitAfter`'s doc:

```suggestion
         * If set to `true`, it bypasses the actionability checks (visible,
```

Reply:
I'm not sure about this option. Playwright has `timeout`, `noWaitAfter` and `strict` but not `force`. Maybe we should remove `force` from the type definitions and docs until we're sure that it's useful?

Reply:
I believe that might be a separate PR. WDYT?

Reply:
I'll remove `force` from this PR, but yeah, to add `strict` will need to be a new PR as we would need to implement the change in the k6 browser module first.

Reply:
Done in 373707517c118664690b3f3f512d454dadcca040, 373707517c118664690b3f3f512d454dadcca040 and 4d0c05f5f714f4206f6fa41de560c0b76402b715